src/sas/sasgui/perspectives/fitting/fitting.py

```diff
@@ -1756,14 +1756,38 @@ def _complete1D(self, x, y, page_id, elapsed, index, model,
                                       data_description="Data unsmeared",
                                       data_id="Data  " + data.name + " unsmeared",
                                       dy=unsmeared_error)
-        # Comment this out until we can get P*S models with correctly populated parameters
         if sq_model is not None and pq_model is not None:
-            self.create_theory_1D(x, sq_model, page_id, model, data, state,
+            # Store which data sets are plotted in which windows
+            # plot_ids[data_id] = [group_id1, group_id2, ...]
+            plot_ids = {}
+            for plot_panel in self.parent.plot_panels.values():
```

Comment:
This seems a bit hacky, is there a better way to check if any plots of the theories exist?